lib/dsn-pcb/dsn-json-to-circuit-json/parse-dsn-to-dsn-json.ts

```diff
@@ -950,32 +959,21 @@ function processVia(nodes: ASTNode[]): Wire {
   if (pathNode?.children) {
     const coords = pathNode.children
       .filter((node) => node.type === "Atom" && typeof node.value === "number")
-      .slice(-2) // Take last two numbers as x,y coordinates
+      .slice(-2) // Take last two numbers as x, y coordinates
 
     if (coords.length === 2) {
-      wire.path = {
-        layer: "all", // vias connect all layers
-        width: 0, // width is defined by the padstack
-        coordinates: coords.map((node) => node.value as number),
-      }
-      wire.type = "via"
-
-      // Find net name if present
-      const netNode = nodes.find(
-        (node) =>
-          node.type === "List" &&
-          node.children?.[0]?.type === "Atom" &&
-          node.children[0].value === "net",
-      )
-      if (netNode?.children?.[1]?.type === "Atom") {
-        wire.net = String(netNode.children[1].value)
-      }
-
-      return wire as Wire
+      wire.path!.coordinates = coords.map((node) => node.value as number)
+    } else {
+      debug("Warning: Missing or incomplete coordinates for via", nodes)
```

Comment:
switch this to an early return pattern, avoid long "if-else" statements, the early return pattern avoids them